tests/draft2020-12/optional/ecmascript-regex.json

```diff
@@ -343,7 +359,11 @@
     },
     {
         "description": "pattern with ASCII ranges",
-        "schema": { "pattern": "[a-z]cole" },
+        "description": "unicode characters do not match ascii ranges",
```

Comment:
There is a duplicate key here, which many JSON decoders will error on.

Am I really the first to run these tests?

Reply:
> Am I really the first to run these tests?

No.  I've run them ([as mentioned](https://github.com/json-schema-org/JSON-Schema-Test-Suite/pull/586#pullrequestreview-1070426493)), but I don't error on duplicate IDs between tests because unit tests should be isolated.

Reply:
There's another duplicate in tests/draft2019-09/vocabulary.json.

(I'm also surprised that CI didn't catch this?)

Reply:
@karenetheridge the JSON Spec allows parsers to decide whether to error on duplicate keys or not, so if the JSON Parser used by CI doesn't error, then it won't catch this.

It's probably better to fix these in a separate PR, as this one is already fairly noisy.

Reply:
@karenetheridge oh wait, it looks like I did this somehow.  I will fix and update.

Reply:
oh... you said "duplicate key" not "duplicate ID".  I retract my statement earlier (but I did run the tests).